src/yz_kv.erl

```diff
@@ -396,6 +405,19 @@ cleanup(_, _) ->
 
 %% @private
 %%
+%% @doc Cleanup tombstones and siblings accordingly... cleanup/3
+%% TODO: make this work for allow_mult=true case
+-spec cleanup([doc()], bkey(), values()) -> [{bkey, bkey()}].
+cleanup([], _BKey, _Values) ->
+    [];
+cleanup([{doc, Fields}|T], BKey, [VH|VT]) ->
```

Comment:
*DO NOT* fix for patch, but I don't think we need Values at all - could just work on the Fields as is if a tombstone, VH will always be <<>>, so it's redundant.